espnet/nets/pytorch_backend/tacotron2/encoder.py

```diff
@@ -52,6 +54,8 @@ def __init__(
 
         Args:
             idim (int) Dimension of the inputs.
+            use_embed (bool, optional) Whether to use embedding table or not.
```

Comment:
Why don't you follow transformer encoder manner?
https://github.com/espnet/espnet/blob/71771245f9ea42a7e8c9ac619c47aa200cc33f6f/espnet/nets/pytorch_backend/transformer/encoder.py#L97-L127